src/plugins/theme/fontSize.js

```diff
@@ -18,6 +18,7 @@ module.exports = function () {
     '23': '2.3rem',
     '24': '2.4rem',
     '25': '2.5rem',
+    '09em': '0.9em',
```

Comment:
This seems like a big outlier. Tell me more about this.

Reply:
From past experience, we use 0.9em quite often for those "slightly smaller than base" font styles. We have 1.4 to 2.5 rem here, then the smaller `em` step we have is `su-text-m0 = 1 em`. I kind of want a `em` one that's just a tad smaller than 1 em.

Reply:
That makes sense but it breaks the naming conventions here. Could this be `-m1` for negative modular 1?

Reply:
I was debating about that since currently the m steps follow the 1.25 ratio, buuuuuut, -m1 is easier to remember and makes sense to not have too many outliers in the class names. So -m1 it is 😄